### PR TITLE
Domestic address livewire component #72 #75 [TGER-133]

### PR DESCRIPTION
**Pull request by devjk1**
do not merge
#72 #75

- added Livewire dependency to composer.json
- created Livewire component & view for DomesticAddressSearchBar
- did not install Tailwind dependency
- did not include save Model DomesticAddress, need to parse results for city and zip FK's.

use
- when the query on _input_ is changed, Service PlacesApi->placeAutocomplete stores results as contacts
- contacts are listed as list-items
- list-item on click the query is updated, and Autocomplete will be called again, rinse and repeat

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
         "dniccum/phone-number": "^1.1",
         "inspheric/nova-email-field": "^1.42",
         "tipoff/authorization": "^2.5.5",
-        "tipoff/support": "^1.7.2"
+        "tipoff/support": "^1.7.2",
+        "livewire/livewire": "^2.0"
```
Comment (drewroberts) [inline composer.json:23]: @pdbreen We are going to be using Livewire in the majority of the packages. Should we add this dependency to tipoff/support?

Comment (pdbreen) [reply composer.json:23]: I'd say if you think most packages will be using it, then sure.  If its just a handful that need it, probably better to leave it as a package pull in.

**Comment by drewroberts** [comment]
@devjk1 In another PR, could you register the livewire component in the package service provider?

I think this is how we do it:

- https://laravel-livewire.com/docs/2.x/package-dev

**Review by drewroberts — APPROVED** [review]
Thank you! 🌁

**Pull request merged**